tensorflow/python/ops/math_ops.py

```diff
@@ -334,9 +334,30 @@ def divide(x, y, name=None):
 
 @tf_export("math.multiply", "multiply")
 @dispatch.add_dispatch_support
-def multiply(x, y, name=None):
-  return gen_math_ops.mul(x, y, name)
 
+def multiply(x, y, name=None):  # pylint: disable=missing-docstring
+  # Do an is comparison here since this is cheaper than isinstance or __eq__
+  """Returns 'x' * 'y' element wise.
+  
+  For example:
+  
+  >>> tf.math.multiply(34, 32) 
+  <tf.Tensor: shape=(), dtype=int32, numpy=1088>
+  
+  
+  Args:
+    'x': A 'Tensor'. Must be one of the following types: 
+    'bfloat16', 'half', 'float32', 'float64', 'uint8', 'int8', 'uint16', 'int16', 'int32', 'int64', 'complex64', 'complex128'.
+    'y': A 'Tensor'. Must have the same type as 'x'.
+    'name': A name for the operation (optional).
+    
+  Returns:
+    A 'Tensor'. Has the same type as 'x'
+  """
+  if y is 1:  # pylint: disable=literal-comparison
```

Comment:
Are you sure you want to make changes in the code as well? 
The title only mentions documentation changes.